roles/girder-install/tasks/main.yml

```diff
@@ -1,45 +1,28 @@
 ---
-
-  - name: apt | cache | update
-    apt: update_cache=yes
-    when: do_install|bool
+  - name: girder | add | node repository
+    shell: "curl -sL https://deb.nodesource.com/setup | sudo bash -"
```

Comment:
I'd recommend to special-case the `nodejs_version` variable so that if it is, say, `"nodesource"`, then nodejs is installed from the nodesource binary.